LICENSES-bundled.txt

```diff
@@ -0,0 +1,7 @@
+This project and source distributions bundle several libraries that are
+compatibly licensed.
+
+
+Name: PyBaMM
```

Comment:
Given that PyBaMM will install some dependencies, do we need to list them here too?

Reply:
I don't think so, because we don't use the source code from those dependencies – it is just PyBaMM.

Reply:
I don't think so, because we don't use the source code from those dependencies – it is just PyBaMM.